components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/GitpodCLIService.kt

```diff
@@ -74,29 +76,35 @@ class GitpodCLIService : RestService() {
                 return "url is missing"
             }
 
-            val resolvedUrl = resolveExternalUrl(url)
-
             return withClient(request, context) { project ->
+                var resolvedUrl = url
```

Comment:
ℹ️ This change can only be tested by hard-coding an URL like "http://localhost:8000" at this moment due to
- https://github.com/gitpod-io/gitpod/issues/11849

Because currently, all URLs come to this function already in the format "https://PORT-(...).gitpod.io".